src/wp-settings.php

```diff
@@ -587,6 +592,23 @@
 	unset( $file );
 }
 
+/**
+ * Adding IndexNow complaint search engine endpoints if enabled.
+ */
+
+if ( defined( 'WP_INDEXNOW' ) && true == WP_INDEXNOW ) {
+	define(
+		'WP_INDEXNOW_PROVIDERS',
```

Comment:
If I understand correctly, it is intended that site admins can override this. However given it is unconditionally declared here (not conditional on providers being defined), it seems like that might not work.